addons/mail/static/src/js/utils.js

```diff
@@ -71,7 +71,7 @@ function linkify(text, attrs) {
 
 function addLink(node, transformChildren) {
     if (node.nodeType === 3) {  // text node
-        const linkified = linkify(node.data);
+        const linkified = linkify(_.escape(node.data));
```

Comment:
I feel like this should be done inside `linkify`, after the link detection, when actually building the tag (both href and url can be wrapped)? But it doesn't solve the other issue I have (see below).

Reply:
yes I think like this:

<https://test.example.com/test?&currency_id=3>

will link on `https://test.example.com/test?&currency_id=3>` I think because > becomes `&gt;` which becomes part of the link

Reply:
@Gorash to answer your latest message, this comment still needs to be resolved before merging I believe.
And also the security team needs to validate any change in these methods due to the sensitivity of them.